content/notes/config.md

```diff
@@ -23,6 +23,40 @@ links: # Links to show in footer
     link: https://github.com/jackyzha0
 ```
 
+### HTML Favicons
+
+If you would like to customize the favicons of your website, you can add them to the `data/config.yaml` file. The default without any set `favicon` key is:
+
+```html
+<link rel="shortcut icon" href="icon.png" type="image/png">
+```
+
+The default can be overridden by defining a value to the `favicon` key in your `data/config.yaml` file. Here is a `List[Dictionary]` example format, which is equivalent to the default:
+
+```yaml
+favicon:
+  - { rel: "shortcut icon", href: "icon.png", type: "image/png" }
+```
+
+In this format, the following keys are available:
+- `rel`: The `rel` attribute of the `<link>` tag.
+- `type`: The `type` attribute of the `<link>` tag.
+- `href` (optional): The `href` attribute of the `<link>` tag.
+- `sizes` (optional): The `sizes` attribute of the `<link>` tag.
+
+If you plan to add multiple favicons generated by a website, it may be easier to define it as HTML:
+
+```yaml
+favicon: |
+  <link rel="shortcut icon" href="icon.png" type="image/png">
```

Comment:
this should also have multiple favicons to match the text above

Reply:
Very good point, fixed 🎯